whois-rpsl/pom.xml

```diff
@@ -24,6 +24,10 @@
             <groupId>commons-codec</groupId>
             <artifactId>commons-codec</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Why not:
```
import java.net.URL;
...
        try {
            new URL("https://geofeed.url");
            // valid
        } catch (MalformedURLException e) {
            // invalid
        }
    }
```

Reply:
Does ```UrlValidator``` do any validation that ```URL``` does not?

Reply:
Using new `URL(value).toURI()`, which should be even more specific in finding invalid URLs, doesn't catch any of the following examples:
- https://notanurl
- https://localhost (which is a valid URL indeed, but I guess it makes no sense in this context)
- https://.com
- ftp://::::@example.com

[Relevant StackOverflow thread](https://stackoverflow.com/questions/1600291/validating-url-in-java#comment78104345_5078838) (but I also tested locally these values, since the thread is quite old I though they might have improved the validation code within `URL`)

Reply:
OK! Thanks. I want to be sure the usefulness of a new dependency outweighs the extra maintenance needed to keep it updated (and the risk that we'll have multiple ways in the code to do a similar thing).